enhancements/cluster-logging/multi-cluster-observability.md

```diff
@@ -26,6 +26,8 @@ tracking-link:
   - https://issues.redhat.com/browse/OBSDA-356
   - https://issues.redhat.com/browse/OBSDA-393
   - https://issues.redhat.com/browse/LOG-4539
+  - https://issues.redhat.com/browse/TRACING-3540
+  - https://issues.redhat.com/browse/OBSDA-489
```

Comment:
This ticket is enough for tracing. The tracing jira above is a child of the OBSDA ticket